apps/ledger-live-desktop/src/newArch/features/ModularDrawer/components/SelectAssetFlow/index.tsx

```diff
@@ -39,8 +43,13 @@ function SelectAssetFlow({ onAssetSelected, currencies }: Readonly<SelectAssetFl
   return (
     <SelectAssetFlowContainer>
       <Header
-        isAssetSelectionVisible={currentStep === FlowStep.SELECT_ASSET_TYPE}
-        currentStep={currentStep}
+        showBackButton={currentStep === FlowStep.SELECT_NETWORK}
```

Comment:
This part is a bit trickier. According to the spec if for example a live app calls the drawer with only currency from the same asset: the drawer opens directly on the network selection and there shouldn't be any back button.

A more common case though is if the drawer is open with a single currency. In this case the drawer should open directly on the account selection and there shouldn't be any back button either.

Sorry I missed the former case in the previous PR

Reply:
Ahh I see. In which case maybe this work should be included in the ticket I'm drafting here: [LIVE-19095](https://ledgerhq.atlassian.net/browse/LIVE-19095)

It feels like all these requirements fall under some kind of unified navigation between flows. I'm at least reluctant to make this PR any bigger. WDYT?